apps/system/utils/netcmd_netmon.c

```diff
@@ -201,8 +205,53 @@ int cmd_netmon(int argc, char **argv)
 			printf("Failed to fetch socket info.\n");
 		}
 	} else if (!(strncmp(argv[1], "wifi", strlen("wifi") + 1))) {
-		/* Get wifi information: SIOCGETWIFI */
-		printf("wifi option will be supported\n");
+#ifdef CONFIG_WIFI_MANAGER
+		wifi_manager_stats_s stats;
+		wifi_manager_info_s info;
+
+		wifi_manager_result_e res = wifi_manager_get_stats(&stats);
+		if (res != WIFI_MANAGER_SUCCESS) {
+			printf("Get Wi-Fi Manager stats failed\n");
+			return ERROR;
+		}
+		printf("=======================================================================\n");
+		printf("CONN    CONNFAIL    DISCONN    RECONN    SCAN    SOFTAP    JOIN    LEFT\n");
+		printf("%-8d%-12d%-11d%-10d", stats.connect, stats.connectfail, stats.disconnect, stats.reconnect);
+		printf("%-8d%-10d%-8d%-8d\n", stats.scan, stats.softap, stats.joined, stats.left);
+		printf("=======================================================================\n");
+
+		printf("Connection INFO.\n");
+		res = wifi_manager_get_info(&info);
+		if (res != WIFI_MANAGER_SUCCESS) {
+			printf("Get Wi-Fi Manager state info failed\n");
```

Comment:
not state info but connection info.

Reply:
Thanks for the comment. Modification completed.